packages/opentelemetry-sdk-trace-base/package.json

```diff
@@ -90,7 +90,7 @@
     "webpack": "5.89.0"
   },
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.0.0 <1.10.0"
+    "@opentelemetry/api": ">=1.9.0 <1.10.0"
```

Comment:
this is a breaking change that we cannot do in a minor version.

Reply:
Any idea to not to make this breaking change or if these changes make sense, do we need to wait for v2?

Reply:
@pichlermarc Maybe this should not be in the API?

Reply:
Does this make sense?
- Move this internal `Console` API to `@opentelemetry/core`
- Keep `diag/consoleLogger` as is (will still access console directly), because `@opentelemetry/api` can't have a dependency to `@opentelemetry/core` so `Console` API is not visible to the `@opentelemetry/api`